schema/units_test.go

```diff
@@ -35,17 +35,22 @@ func TestUnitsParseInt(t *testing.T) {
 	}
 
 	for testCase, testData := range testMatrix {
+		// When executed in parallel, referencing testData from the
+		// outer scope will not produce the proper value, so we need
+		// to bind it to a variable, localTestData, scoped inside
+		// the loop body.
+		localTestData := testData
 		t.Run(testCase, func(t *testing.T) {
```

Comment:
Do we need/want a call to `t.Parallel()` before the `for` statement?

Reply:
Adding `units_test` to the parallel execution step, creates a data race between `units_test` and the other parallelized tests.

Reply:
Will address in another issue.

Reply:
The answer to my original question as posed is "no".

However, there probably _should_ be a `t.Parallel()` added inside the anonymous function at line 43.

Reply:
Given our new-found understanding of `t.Parallel()`, that call should be added to the functions invoked via `t.Run()` and it should not be added to the base test functions.

And, since adding that call here triggers data race errors (due to the lazy evaluation of the units regular expression cache), adding it should be deferred until we have an approach to addressing the race (see #76); whether that should be handled by this PR or a follow-on is an orthogonal question.